src/main/java/htsjdk/variant/variantcontext/VariantContext.java

```diff
@@ -1675,18 +1678,14 @@ public String getContig() {
      * </p>
      *
      * <p>
-     *     INDEL events usually start on the first unaltered reference base before the INDEL.
-     * </p>
-     *
-     * <p>
+     *     <strong>Warning:</strong>
      *     Note also that the VCF spec allows 0 and N + 1 for POS field for telomeric event,
      *     where N is the length of the chromosome.
      *     The "0" value returned should be interpreted as telomere, and does not violate the above "1-based" comment.
-     *     It's the responsibility of code generating such variants to make sure {@code start} is populated correctly.
```

Comment:
Is unfortunate the the spec talks about "Telomeres" only. It is assuming that the CHROM (thus the referene) only can contain full non-circular chromosomes.... total BS. (e.g. the MT chromosome in humans or nearly every contig in unfinished genomic references) 
Yes CHROM as a name wasn't a good choice to start with but we don't need to keep up to that mistake. Notice that API using "contig" instead of "chr" or "chromosome; it is still assuming too much but is closer to the truth..  

So I would refrain to make it seems as it can only be telomerees.

What about something like:

"
This property can take on "0" and "N+1" (where N is the last base in the enclosing contig) when this variant record makes references to events that happen before or at the beginning or after or at the end of the enclosing contig."

Reply:
I hope this PR does not turn into a loath about the spec itself. I can only work within the current spec for this PR.
So, since the latest spec talks about telomere, I'm happy to use a single word "telomere" to avoid such long sentence.

Reply:
It won't, I leave it up to you at this point.